BGL/include/CGAL/boost/graph/named_params_helper.h

```diff
@@ -112,8 +113,156 @@ namespace CGAL {
     typedef typename boost::property_traits<PMap>::value_type type;
   };
 
+  namespace internal_np
+  {
+  template<typename key_value, class PMap, class Graph, typename Tag>
+  struct MapInitializer{
```

Comment:
you must respect CGAL naming conventions, something like `Index_map_initializer`